application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -241,7 +241,7 @@
   ## This is needed to update the number of user when the status is active and the deleted users are hidden.
   #set ($usersCountActive = 0)
   #foreach ($obj in $doc.getObjects('XPoll.XPollVoteClass'))
-    #if ($xwiki.exists($obj.user))
+    #if ($xwiki.exists($services.model.resolveDocument($user)) || $xwiki.exists("$xcontext.mainWikiName:$user"))
```

Comment:
This is not right. The local user and the global user are different users. The user reference saved when voting should contain the information needed to identify the voter. See my comment above regarding "local" vs. "compactwiki" serializer.